app/assets/javascripts/angular/resources/shared_graph_behavior.js

```diff
@@ -21,14 +21,15 @@ angular.module("Prometheus.services").factory("SharedGraphBehavior", ["$http", "
       $scope.widgets = $scope.widgets || [];
       var urlVars = URLVariablesDecoder();
       var templateVarRe = /^var\.(.*)$/;
-      for (var o in urlVars) {
-        var matches = o.match(templateVarRe)
+      var o;
```

Comment:
Just wondering: do you know why it suggests this change? IMO it makes the code less readable because now you need to assume that `o` is used elsewhere than just in the loop.